### PR TITLE
#7961: Update logical or doc and sweep config

### PR DESCRIPTION
**Pull request by umadevimcw**
### Ticket
#7961 

### Problem description
Documentation was not updated with supported configs

### What's changed
Updated the documents and sweep test configs
Moved old sweep to new sweep

### Checklist
- [x] [Post commit CI passes](https://github.com/tenstorrent/tt-metal/actions/runs/11068525956)
- [x] [TTNN sweep CI](https://github.com/tenstorrent/tt-metal/actions/runs/11128727874) - All the sweep test files updated in this PR are passed (logical ops)

### CODE REVIEW

**Review by Aswinmcw — APPROVED** [review]
LGTM

**Review by VirdhatchaniKN — APPROVED** [review]

**Review by mcw-anasuya — APPROVED** [review]

**Review by eyonland — CHANGES_REQUESTED** [review]
These tests are deprecated.  They are being moved to the new framework sweeps location.

**Comment by umadevimcw** [comment]
> These tests are deprecated. They are being moved to the new framework sweeps location.

Updated

**Review by jdesousa-TT — APPROVED** [review]

**Comment by jdesousa-TT** [comment]
Please run the new sweeps on the `ttnn - run sweeps` pipeline before merging.

**Review by eyonland — APPROVED** [review]

**Review by ayerofieiev-tt — APPROVED** [review]

File: ttnn/cpp/ttnn/operations/eltwise/binary/binary_pybind.hpp [hunk 1]
```diff
@@ -105,6 +105,9 @@ void bind_binary_operation(py::module& module, const binary_operation_t& operati
 
         Supports broadcasting.
 
+        Note:
```
Comment (cfjchu) [inline ttnn/cpp/ttnn/operations/eltwise/binary/binary_pybind.hpp:108]: what is this?

Comment (umadevimcw) [reply ttnn/cpp/ttnn/operations/eltwise/binary/binary_pybind.hpp:108]: This section will have information like supported types, layouts etc.. which will be passed as one of the parameter

**Pull request merged**